src/Containers/Project/Project.js

```diff
@@ -8,17 +11,44 @@ export class Project extends Component{
       expanded: false
     }
   }
+  
+  fetchPalettes = async () => {
+    const { id, currentExpandedProject } = this.props
+    if(id === currentExpandedProject) {
+      this.props.setCurrentExpandedProject('')
+    } else {
+      this.props.setCurrentExpandedProject(id)
+    }
+    if(!this.props.palettes.length){
+      await this.props.fetchProjectPalettes(id)
+    }
+  }
+
   render() {
+    const { name, palettes, id, currentExpandedProject } = this.props
     return(
       <div>
-        <h3>Project Name</h3>
+        <div onClick={this.fetchPalettes}>
+          <p>{name}</p>
+          <i className="fas fa-pencil-alt"></i>
+          <i className="far fa-trash-alt"></i>
+        </div>
         {
-          this.state.expanded &&
-          <MiniPaletteArea/>
+          id ===  currentExpandedProject &&
```

Comment:
Nice work with the check here if it is the currentExpandedProject! This looks really clean 🙂